app/components/Footer.tsx

```
@@ -0,0 +1,36 @@
+  return (
+    <footer className="bg-gray-900 text-white py-8 mt-auto">
+      <div className="container mx-auto px-4">
+        <div className="grid grid-cols-1 md:grid-cols-3 gap-8">
+          <div>
+            <h3 className="text-xl font-bold mb-4">HostelHub</h3>
+            <p className="text-gray-400">Find your perfect student accommodation with ease.</p>
+          </div>
+          <div>
+            <h4 className="text-lg font-semibold mb-3">Quick Links</h4>
+            <ul className="space-y-2">
+              <li><a href="/" className="text-gray-400 hover:text-white transition-colors">Home</a></li>
```

ESLint

[error] Do not use an `<a>` element to navigate to `/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)